### DISCUSSION TITLE
Add doc section of used WB peripherals

### DISCUSSION THREAD

**Discussion by andkae** [stnolting/neorv32#535] [Ideas]
Hi Stephan,

one idea which come to my mind when i added the IP recommendation in #533. Perhaps could be a good idea to have a subpage in the docu where the used Wishbone Peripherals with the NEORV32 are documented. That would relax the external IP selection for other users. For instance i use [IICMB](https://github.com/sshuv/iicmb) as I2C controller and it works well.

BR,
Andreas

**Comment by stnolting** [comment]
Hey there!

I also thought about this, but having a list of external modules right inside the documentation might look like "hey, this is officially approved!" - which it isn't 😅 So maybe it would be better to have a new thread in the discussion section where users can list Wishbone components (like the ones you have mentioned) that work out of the box when being attached to the processor.

Or maybe we can have a list somewhere in neorv32-setups?! 🤔 I'm not sure... But I'm open for discussion.

**Reply by vhdlnerd** [reply]
I vote for having a list somewhere in the repo (neorv32-setups, maybe, since it already has external links to other repos). And, also have a thread in the discussion section announcing new WB cores.  i personally like to peruse a concise list instead of having to wade through 100s of posts to find something.
(Unless you can make a discussion post "sticky" that could contain the list of WB cores.)

**Reply by andkae** [reply]
> but having a list of external modules right inside the documentation might look like

Mhh also a good point :) And if this goes not in the online documentation and also in no pdf - It's only a subpage on github. I like the idea with [neorv32-setups](https://github.com/stnolting/neorv32-setups) because it not messes the neorv32 main repo up. Perhaps there could be a section WB Peripherals connected to:

A table prototype could look as follows:

| Name                                                        | Type           | Integrated by |
| -------------------------------------------- | ------------ | --------------- |
| [IICMB](https://github.com/sshuv/iicmb) | I2C Master | @andkae       |

**Reply by stnolting** [reply]
Looks good to me! But I am not really sure where to put this?! 🤔 A new section in the README of neorv32-setup? Or maybe it would be better to have a new discussion with a catchy title and post all our "findings" there.
